static/assets/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/mstile-150x150.png"/>
```

Comment:
Given that you've already added tile images for the other sizes, should we not reference them here?

Syntax is described here: https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/dn320426(v=vs.85)

Reply:
no idea why these weren't generated in the first place 😛 anyway, done now